### ISSUE TITLE
Node Shape

### ISSUE THREAD

**Issue by andre-msantos** [kayjan/bigtree#36]
Hi Kay!

Congratulations! Your library is awesome.

Could you help me? I've never used Graphviz before, and I'm having trouble changing the node's shape so that I can save it as an image. 

I tried:

`Node("a", age=90, shape='diamond')`
`graph = tree_to_dot(root, node_colour='#0099cc', node_attr='{"shape":"diamond"}')`

But these codes didn't work.

Thank you

**Comment by kayjan** [comment]
Hello, thanks for using bigtree!

There is a similar example in the documentation under the tips and trick section [here](https://bigtree.readthedocs.io/en/latest/others/tips.html#trees-with-weighted-edges), but the example is for adding edge attributes - for your case it would be to add node attributes.

Below is a sample code on how to have a mixture of DiamondNode and regular Node. I realize there was some bug in the code (thanks to your issue!), I have a fix in bigtree v0.6.8, do update the bigtree package before running the sample code below.

```
from bigtree import Node, tree_to_dot


class DiamondNode(Node):
    def __init__(self, name, **kwargs):
        super().__init__(name, **kwargs)

    @property
    def node_attr(self):
        return {"shape": "diamond"}


# Construct tree
root = DiamondNode("a")
b = Node("b", parent=root)
c = Node("c", parent=root)
d = Node("d", parent=b)

graph = tree_to_dot(root, node_attr="node_attr")
graph.write_png("diamond_tree.png")
```

This will result in the output,
![diamond_tree](https://user-images.githubusercontent.com/29906371/212470747-f7c56a8a-21b7-4ec5-a20c-d6d5041ef575.png)

Hope this helps!

**Comment by kayjan** [comment]
**Update**: In bigtree v0.7.2 I added the `node_shape` parameter to quickly customize the node shape.

```python
from bigtree import Node, tree_to_dot

root = Node("a")
b = Node("b", parent=root)
c = Node("c", parent=root)
d = Node("d", parent=b)

graph = tree_to_dot(root, node_shape="diamond")
graph.write_png("diamond_tree.png")
```

This will result in the output,
![diamond_tree](https://user-images.githubusercontent.com/29906371/219875620-8686fd98-07e3-4231-984e-e71e1f1245c4.png)

Hope this helps!
